components/automate-ui/src/app/entities/layout/layout-sidebar.service.ts

```diff
@@ -1,278 +1,289 @@
+            }],
+            compliance: [{
+                name: 'Compliance',
                 items: [
                     {
-                        name: 'Users',
-                        icon: 'person',
-                        route: '/settings/users',
+                        name: 'Reports',
+                        icon: 'equalizer',
+                        route: '/compliance/reports',
                         authorized: {
-                            name: 'users',
-                            allOf: ['/auth/users', 'get']
+                            name: 'reports',
+                            allOf: [['/compliance/reporting/stats/summary', 'post'],
+                            ['/compliance/reporting/stats/failures', 'post'],
+                            ['/compliance/reporting/stats/trend', 'post']]
                         },
                         visible: true
                     },
                     {
-                        name: 'Teams',
-                        icon: 'people',
-                        route: '/settings/teams',
+                        name: 'Scan Jobs',
+                        icon: 'wifi_tethering',
+                        route: '/compliance/scan-jobs',
                         authorized: {
-                            name: 'teams',
-                            allOf: ['/auth/teams', 'get']
+                            name: 'reports',
+                            allOf: [['/compliance/scanner/jobs', 'post'],
+                            ['/compliance/scanner/jobs/search', 'post']]
                         },
                         visible: true
                     },
                     {
-                        name: 'API Tokens',
-                        icon: 'vpn_key',
-                        route: '/settings/tokens',
+                        name: 'Profiles',
+                        icon: 'library_books',
+                        route: '/compliance/compliance-profiles',
                         authorized: {
-                            name: 'tokens',
-                            anyOf: [['/auth/tokens', 'get'],
-                                ['/iam/v2beta/tokens', 'get']]
+                            name: 'profiles',
+                            allOf: [['/compliance/profiles/search', 'post']]
                         },
                         visible: true
                     }
                 ],
                 visible: true
-            },
-            {
-                name: 'Access Management',
-                items: [
-                    {
-                        name: 'Policies',
-                        icon: 'security',
-                        route: '/settings/policies',
-                        authorized: {
-                            name: 'policies',
-                            allOf: ['/iam/v2beta/policies', 'get']
+            }],
+            settings: [
+                {
+                    name: 'Node Management',
```

Comment:
Have not dug in far enough yet, but do you have a mechanism to display/hide this group name based on the collection of anyOf/allOf of its children?

Reply:
@msorens good point! I'll add that logic.